### DISCUSSION TITLE
Change the title of the Bonsai window

### DISCUSSION THREAD

**Discussion by alexpiet** [bonsai-rx/bonsai#1649] [General]
When I start Bonsai with my_workflow.bonsai, the title of the Bonsai window becomes "my_workflow.bonsai (Running) - Bonsai"

I would like to modify this title to display more useful information. How can I modify this title?

**Comment by glopesdev** [comment]
@alexpiet there is currently no way to do this, but I guess you could get the current process window handle and set this title using native OS apis. Note that this would always be a platform-specific solution, especially if you are running this from the command line.

What kind of information were you hoping to see and how would it find its way there?

**Reply by alexpiet** [reply]
Thanks for the fast response. 

The information I would like to display is what rig is being controlled by Bonsai. This information would be available when Bonsai is launched, and in a settings file that is being passed to Bonsai. 

This is always being run on a Windows OS, so the platform-specific solution should work for my use case. I will give it a try
